test/models/image_script_test.rb

```diff
@@ -174,39 +174,17 @@ def mysql_config_file
     script = script_file("retransfer_images")
     tempfile = Tempfile.new("test").path
 
-    File.open("#{local_root}/orig/#{in_situ_id}.tiff", "w") do |f|
-      f.write("A")
-    end
-    File.open("#{local_root}/orig/#{in_situ_id}.jpg", "w") do |f|
-      f.write("B")
-    end
-    File.open("#{local_root}/1280/#{in_situ_id}.jpg", "w") do |f|
-      f.write("C")
-    end
-    File.open("#{local_root}/960/#{in_situ_id}.jpg", "w") do |f|
-      f.write("D")
-    end
-    File.open("#{local_root}/640/#{in_situ_id}.jpg", "w") do |f|
-      f.write("E")
-    end
-    File.open("#{local_root}/320/#{in_situ_id}.jpg", "w") do |f|
-      f.write("F")
-    end
-    File.open("#{local_root}/thumb/#{in_situ_id}.jpg", "w") do |f|
-      f.write("G")
-    end
-    File.open("#{local_root}/960/#{turned_over_id}.jpg", "w") do |f|
-      f.write("H")
-    end
-    File.open("#{local_root}/640/#{turned_over_id}.jpg", "w") do |f|
-      f.write("I")
-    end
-    File.open("#{local_root}/320/#{turned_over_id}.jpg", "w") do |f|
-      f.write("J")
-    end
-    File.open("#{local_root}/thumb/#{turned_over_id}.jpg", "w") do |f|
-      f.write("K")
-    end
+    File.write("#{local_root}/orig/#{in_situ_id}.tiff", "A")
+    File.write("#{local_root}/orig/#{in_situ_id}.jpg", "B")
+    File.write("#{local_root}/1280/#{in_situ_id}.jpg", "C")
+    File.write("#{local_root}/960/#{in_situ_id}.jpg", "D")
+    File.write("#{local_root}/640/#{in_situ_id}.jpg", "E")
+    File.write("#{local_root}/320/#{in_situ_id}.jpg", "F")
+    File.write("#{local_root}/thumb/#{in_situ_id}.jpg", "G")
+    File.write("#{local_root}/960/#{turned_over_id}.jpg", "H")
+    File.write("#{local_root}/640/#{turned_over_id}.jpg", "I")
+    File.write("#{local_root}/320/#{turned_over_id}.jpg", "J")
+    File.write("#{local_root}/thumb/#{turned_over_id}.jpg", "K")
```

Comment:
Nice!  I wish I'd known about File.write() from the start.  How much cleaner and more compact!